core/internal/helpers/sarama.go

```diff
@@ -523,3 +524,27 @@ func (m *MockSaramaPartitionConsumer) HighWaterMarkOffset() int64 {
 	args := m.Called()
 	return args.Get(0).(int64)
 }
+
```

Comment:
Could we verify why the checks have been failing for continuous-integration on travis-ci build?